super-intel.c

```diff
@@ -645,6 +645,14 @@ struct imsm_update_rwh_policy {
 	int dev_idx;
 };
 
+enum imsm_sku {
+	SKU_NO_KEY = 0,
+	SKU_STANDARD_KEY,
```

Comment:
In this case enum values are meaningful, please keep assignments for every enum value, do not let compiler or developer to guess.